packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -0,0 +1,52 @@
+const { Command } = require('@oclif/command');
+
+const { startSpinner, endSpinner } = require('../utils/display');
+
+const inquirer = require('inquirer');
+
+class ZapierBaseCommand extends Command {
+  run() {
+    this.parseFlags();
+    return this.perform();
```

Comment:
So I agree and I like that it would be explicit in the command. That'll also let us not replicate [these lines](https://github.com/zapier/zapier-platform/pull/43/files#diff-569f6a006df4c3df05da24f1dbb24b03R28-R29) in every command.

The thing that attaching them to the instance gets us is that `this.log('neat')` can safely read `this.flags.format` (instead of calling it like `this.log('neat', flags.format)`).

We could put it in both places (both `this.flags` and `flags` argument), but then I worry that it'll be too easy for them to get out of sync.

Given that 

```js
const {x, y, z} = this.flags
```

is a very common js pattern in react (one we use [here](https://github.com/zapier/zapier/blob/7fd99973ab048e75455482df6d6a8cf2f32eadab/web/frontend/assets/app/developer-v3/components/AppList/Item.js#L37), for instance) and it buys us a lot of convenience in base util functions, I lean towards keeping it as is.

Reply:
Sounds fair! 👍